docs/features/job-queue.txt

```diff
@@ -4,63 +4,136 @@
+observe all submitted jobs and their states, and provides the ability to modify some configuration
+options, like priority, position in the queue, and resource pool.
+
+************
+ Job States
+************
+
+Jobs in the Job Queue can be in the ``Queued`` or ``Scheduled`` states. A ``Queued`` job has been
+received by Determined but has not been allocated resources yet. Any change made to queued jobs
+should take effect as soon as they have been received by Determined. A job in the ``Scheduled``
+state has already been chosen by the scheduler to start execution and it may have already acquired
+resources or be running at that moment. Once it has started running, a progress bar like the one
+seen in the experiments section should appear. A job that has already started to run will not be
+able to accept changes immediately and instead needs to be preempted.
+
+Jobs that have completed or errored out are not counted as active and are omitted from this list.
 
 ***********************
  Visualizing Job Queue
 ***********************
 
-Queue Management's job queue can be viewed from the CLI or from the WebUI. To view it in the WebUI,
-click on the `Job Queue` tab. To view it in the CLI, use the following command:
+The Job Queue can be viewed from the CLI or from the WebUI. To view it in the WebUI, click on the
+`Job Queue` tab. To view it in the CLI, use either of the following commands:
 
-.. code:: bash
+.. code::
 
-   det job list
+   $ det job list
+   $ det job ls
 
-Both the WebUI and the CLI will display a table of results, ordered by scheduling order. Note that
-the scheduling order is not necessarily the priority of the job. In addition to the order of the
-job, users can see the state of the jobs and how many slots each job is using.
+The above commands show the queue of the default resource pool. To view the queue of other resource
+pools, specify the ``--resource-pool`` option:
 
-************
- Job States
-************
+.. code::
 
-Jobs can be in the Queued or Scheduled states. Queued jobs have been received by Determined but have
-not been allocated resources yet. Any change made to those experiments in the Job Queue should take
-effect as soon as they are received. A job in the Scheduled state has selected to run by the
-scheduler; it may or may not have acquired resources or be running at that moment. Once it has
-started running, a progress bar like the one in the experiments tab should appear. A job that has
-already started to run will not reflect changes immediately and will instead need to be preempted.
+   $ det job list --resource-pool compute
 
-Jobs that have completed or errored out are not counted as active and are omitted from this list.
+For more information about the CLI options, see the CLI documentation or use ``det job list -h``.
+
+Both the WebUI and the CLI will display a table of results, ordered by scheduling order. Note that
```

Comment:
something like
.. by the scheduling order _if available_ (eg supported by the scheduler type)